src/components/HdToggle.vue

```diff
@@ -29,13 +48,16 @@
 <script>
 import OnResizeService from 'homeday-blocks/src/services/on-resize';
 import HdIcon from 'homeday-blocks/src/components/HdIcon.vue';
+import HdButton from 'homeday-blocks/src/components/buttons/HdButton.vue';
 import { chevron as chevronIcon } from 'homeday-assets';
+import kebabMenuIcon from '@/assets/kebab-menu.svg';
```

Comment:
Don't we want to add this to hd-assets?
If we ever reuse it, would be very helpful to have it there.

Reply:
```suggestion
import { kebabMenu as kebabMenuIcon } from 'homeday-assets'
```
@petterHD a new `homeday-assets` version was published and it has this new kebab menu icon.
Could you please update `homeday-assets` to version `4.56.0`?